src/main/java/com/epam/gym/controller/AuthController.java

```diff
@@ -21,7 +21,7 @@
 public class AuthController {
     private final UserService userService;
 
-    @GetMapping("/login")
+    @PostMapping("/login")
     public ResponseEntity<String> login(@Valid @RequestBody UserCredentials credentials) {
         log.info("Login attempt for user: {}", credentials.username());
```

Comment:
I think debug level is more appropriate here:
```suggestion
        log.debug("Login attempt for user: {}", credentials.username());
```